webapp/pages/profile/_id/_slug.vue

```diff
@@ -89,65 +89,19 @@
-          <ds-space v-if="user.following && user.following.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.following') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.following && user.following.length">
-            <ds-space v-for="follow in uniq(user.following)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followingCount - user.following.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followingCount - user.following.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followingNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :loading="$apollo.loading"
+          :user="user"
+          type="followedBy"
+          @fetchAllConnections="fetchAllConnections"
+        />
         <ds-space />
-        <base-card style="position: relative; height: auto;">
-          <ds-space v-if="user.followedBy && user.followedBy.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.followedBy') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.followedBy && user.followedBy.length">
-            <ds-space v-for="follow in uniq(user.followedBy)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followedByCount - user.followedBy.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followedByCount - user.followedBy.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followedByNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :loading="$apollo.loading"
+          :user="user"
+          type="following"
+          @fetchAllConnections="fetchAllConnections"
```

Comment:
> [<img alt="alina-beck" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/alina-beck) **Authored by [alina-beck](https://github.com/alina-beck)**
_<time datetime="2020-04-07T14:16:41Z" title="Tuesday, April 7th 2020, 4:16:41 pm +02:00">Apr 7, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -89,65 +89,19 @@
         <ds-heading tag="h3" soft style="text-align: center; margin-bottom: 10px;">
           {{ $t('profile.network.title') }}
         </ds-heading>
-        <base-card style="position: relative; height: auto;">
-          <ds-space v-if="user.following && user.following.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.following') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.following && user.following.length">
-            <ds-space v-for="follow in uniq(user.following)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followingCount - user.following.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followingCount - user.following.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followingNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :user="user"
+          type="followedBy"
+          @fetchAllConnections="fetchAllConnections"
+          :loading="$apollo.loading"
+        />
         <ds-space />
-        <base-card style="position: relative; height: auto;">
-          <ds-space v-if="user.followedBy && user.followedBy.length" margin="x-small">
-            <ds-text tag="h5" color="soft">
-              {{ userName | truncate(15) }} {{ $t('profile.network.followedBy') }}
-            </ds-text>
-          </ds-space>
-          <template v-if="user.followedBy && user.followedBy.length">
-            <ds-space v-for="follow in uniq(user.followedBy)" :key="follow.id" margin="x-small">
-              <!-- TODO: find better solution for rendering errors -->
-              <client-only>
-                <user-teaser :user="follow" />
-              </client-only>
-            </ds-space>
-            <ds-space v-if="user.followedByCount - user.followedBy.length" margin="small">
-              <ds-text size="small" color="softer">
-                {{
-                  $t('profile.network.andMore', {
-                    number: user.followedByCount - user.followedBy.length,
-                  })
-                }}
-              </ds-text>
-            </ds-space>
-          </template>
-          <template v-else>
-            <p style="text-align: center; opacity: 0.5;">
-              {{ userName }} {{ $t('profile.network.followedByNobody') }}
-            </p>
-          </template>
-        </base-card>
+        <follow-list
+          :user="user"
+          type="following"
+          @fetchAllConnections="fetchAllConnections"
+          :loading="$apollo.loading"
```


I know @mattwr18 pointed this out already and you marked it as resolved – but `:loading` should go above `@fetchAllConnections`, also for the `follow-list` above. :wink:

Reply:
> [<img alt="rbeer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/rbeer) **Authored by [rbeer](https://github.com/rbeer)**
_<time datetime="2020-04-09T21:17:27Z" title="Thursday, April 9th 2020, 11:17:27 pm +02:00">Apr 9, 2020</time>_

---


2c7ca87ac